src/package.lisp

```diff
@@ -21,5 +21,11 @@
            #:serialize
            #:deserialize
            #:clear-messages
-           #:python-message-spec))
-
+           #:python-message-spec
+           ;; RPC server
+           #:make-dispatch-table
```

Comment:
See note about add'l exports.

Reply:
Exporting the error types and their accessors? I did that; I added a comment to try to make it clearer.

Reply:
whoops, maybe i didn't see